rpyscope/gui_qt.py

```diff
@@ -216,7 +223,6 @@ def __init__(self):
 
         self.path_button = QPushButton("Browse")
         self.path_button.clicked.connect(self.set_path)
-        self.path_button.acceptDrops()
```

Comment:
Why is this removed? I think it allows a user to select a path by simply dragging a file onto the button... might be good to check.

Reply:
acceptDrops tells the system that the push button may be able to accept drop events, but I didn't write any function to handle this event.  I don't know why I wrote this line in the first place. To me this looks unnecessary, but if you really like it I can implement it, but it makes more sense to me to drag onto the text input.